packages/world/src/systems/init/InitPlayersSystem.sol

```diff
@@ -4,15 +4,31 @@ pragma solidity >=0.8.24;
 import { System } from "@latticexyz/world/src/System.sol";
 
 import { ObjectTypeMetadata, ObjectTypeMetadataData } from "../../codegen/tables/ObjectTypeMetadata.sol";
-
+import { ObjectTypeSchema } from "../../codegen/tables/ObjectTypeSchema.sol";
+import { MAX_PLAYER_ENERGY } from "../../Constants.sol";
 import { PlayerObjectID } from "../../ObjectTypeIds.sol";
 
 contract InitPlayersSystem is System {
   function initPlayerObjectTypes() public {
     // Players
     ObjectTypeMetadata._set(
       PlayerObjectID,
-      ObjectTypeMetadataData({ stackable: 0, maxInventorySlots: 36, mass: 10, energy: 10, canPassThrough: false })
+      ObjectTypeMetadataData({
+        stackable: 0,
+        maxInventorySlots: 36,
+        mass: 10,
+        energy: uint32(MAX_PLAYER_ENERGY),
+        canPassThrough: false
+      })
     );
+
+    int32[] memory relativePositionsX = new int32[](1);
```

Comment:
not sure how this would look, but as schemas don't change, would it be better to have fixed schemas instead (not reading from storage)?

Reply:
yea that would save gas, so we'd read this from a library?

Reply:
could be done inline really, or have a function that returns the schemas for each object